requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy # need to install first otherwise dependency issues
+numpy  # need to install first otherwise dependency issues
```

Comment:
Is this extra space needed?

Reply:
[flake8 rule](https://lintlyci.github.io/Flake8Rules/rules/E261.html)